.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: go
+
+go:
+  - 1.5
```

Comment:
It looks like travis does weird stuff with tip.
For Vault despite mentioning tip, Travis is picking up 1.4.1.
I guess it is better to explicitly mention 1.6, now that it is released.